source/developers-guide/best-practise/codestyle-guide/index.md

```diff
@@ -0,0 +1,370 @@
+---
+layout: default
+title: Shopware Codestyle Guide
+github_link: developers-guide/codestyle-guide/index.md
+tags:
+  - api
+indexed: true
+---
+
+Our coding standard is based on the standards of Typo3 and Zend-Framework. A quick summary of the guide can be downloaded [here](http://community.shopware.com/files/downloads/swagcodingstandard-661541.pdf).
```

Comment:
Code standards are PSR-1 and PSR-2: https://github.com/shopware/shopware/blob/5.1/CONTRIBUTING.md

Remove the link to the file